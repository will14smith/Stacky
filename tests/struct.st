struct IntPair { a i64 b i64 }

main () -> () { @IntPair 1 ~a 2 ~b sum print drop }

sum IntPair -> IntPair i64 { dup dup #a swap #b + }
