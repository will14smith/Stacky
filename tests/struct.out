=== Output ===
3

=== Exit ===
0
